enhancements/aiops/20230216-enable-gpu-services-training.md

```diff
@@ -0,0 +1,42 @@
+# Title: 
+Enable GPU Services upon submission of initial training job for workload logs.
+
+## Summary: 
+Currently, when a user would like to enable AI services, they will go to the Opni Admin Dashboard. First, they are required to enable Logging and once that is done, they will go to the AIOps panel and check the "Enable GPU Services button". When they hit the "Save" button, the GPU services are installed within the Kubernetes cluster. This includes the workload DRAIN service, the training controller service, the GPU Controller service and CPU Inferencing service. However, this feature can be a little more straight forward by simply detecting the existence of a GPU within the cluster and when the user decides to train a Deep Learning model, that is when these GPU services should be installed, rather than through a checkbox button.
```

Comment:
- [ ] Rephrase `However, this feature can be a little more straight forward` to `This UX can be avoided...`
- [ ] Can't be `the existence of a GPU within the cluster`. Must be `the availability of a GPU within the cluster`
- [ ] Rephrase `user decides to train a Deep Learning model` to `user creates/updates the workload log anomaly watchlist`